components/scripts/maven/mapping.example

```diff
@@ -1,9 +0,0 @@
-# This file serves as an example of how to define a custom mapping for the names of the custom values
-# that are extracted from the build scans that are fetched when running experiment 03 and 04.
-#
```

Comment:
By sharing the mapping file between Gradle and Maven, the text (which is currently build tool specific) has to become build tool agnostic.

```
...that are extracted from the build scans that are fetched when running the experiments that involve builds from CI.
```

Reply:
✅ Done